test/sanity/ansible-doc/skip.txt

```diff
@@ -40,3 +40,4 @@ panos_security_policy
 s3
 vsphere_guest
 win_msi
+apconos_config
```

Comment:
New modules should not be added to the skip list.

Reply:
I removed it.
Thanks!

Reply:
> 
> 
> New modules should not be added to the skip list.

I removed it.
Thanks!